src/components/SendFundsForm.vue

```diff
@@ -527,7 +527,14 @@ export default {
           () =>
             isErc20(this.currency)
               ? this.ethBalance >= this.transferFee || this.$t('transfer.error_not_enough_eth_fee')
-              : true
+              : true,
+          (v) => {
+            const isKlyTransfer = this.currency === Cryptos.KLY
+            if (!isKlyTransfer) return true
+            const MAX_UINT64 = BigInt('18446744073709551615')
+            const isKlyTransferAllowed = isKlyTransfer && this.transferFee && v < MAX_UINT64
```

Comment:
1. Redundant check on `isKlyTransfer `. You already used early return on line 533.

2. What's the purpose on checking `this.transferFee`?

3. I think `MAX_UINT64` is represented in Beddows, while the amount (`v`) is represented in KLY. If so, you must first convert `v` into Beddows and then compare it with `MAX_UINT64`. Check `convertklyToBeddows` helper.

Reply:
1 KLY = 100,000,000 Beddows

Reply:
2. "Handle the runtime error when estimating [fee]. Return 0 fee if error catched" In normal cases this.transferFee cannnot be zero.